src/Domain/Donation.php

```diff
@@ -442,6 +474,36 @@ public function setAmount(string $amount): void
         $this->amount = $amount;
     }
 
+    /**
+     * @param string $psp
+     * @param string $cardBrand
+     * @param string $cardCountry - default to 'GB' on create, an update command will override this where necessary
+     * @param string $charityFee
+     */
+    public function setCharityFee(string $psp, string $cardBrand = '', string $cardCountry = 'GB'): void
+    {
+        $feeAmountFixed = '0.20';   // 20p fixed per-donation
+
+        if ($psp === 'enthuse') {
+            $feeRatio = '0.012';
```

Comment:
Should be 0.019 – we used 1.2% in the old method because Stripe data was its only use case

Reply:
@NoelLH updating this to '0.019' seems to be breaking some irrelevent tests, i'm not 100% sure why?! Any ideas from this output log:

```
MatchBot\Tests\Application\Actions\Donations\CreateTest::testSuccessWithStripeAccountIDMissingInitiallyButFoundOnRefetch
TypeError: Argument 1 passed to MatchBot\Domain\Donation::setClientSecret() must be of the type string, null given, called in /var/www/html/src/Application/Actions/Donations/Create.php on line 151
```
and 

```
MatchBot\Tests\Application\Actions\Donations\CreateTest::testSuccessWithMatchedCampaignUsingStripe
TypeError: Argument 1 passed to MatchBot\Domain\Donation::setClientSecret() must be of the type string, null given, called in /var/www/html/src/Application/Actions/Donations/Create.php on line 151
```